packages/cli/src/create/create.index.ts

```diff
@@ -0,0 +1,13 @@
+import { TarReader } from '@cotar/core';
+import { createWriteStream, promises as fs } from 'fs';
+import type { Logger } from 'pino';
```

Comment:
Do we need to update the Logger as LogeType?

Reply:
Nope, this is the `cli` only we should not be calling this function directly.

If we want to programatically create a tar index we should use `TarReader.index`